app/bundles/ReportBundle/Model/ReportModel.php

```diff
@@ -733,21 +733,30 @@ private function getOperatorOptions(array $data)
     }
 
     /**
-     * @return int
+     * Get the total of rows found by the query. Ignore pagination limits.
      */
-    private function getTotalCount(QueryBuilder $qb, array &$debugData)
+    private function getTotalCount(QueryBuilder $qb, array &$debugData): int
     {
         $countQb = clone $qb;
-        $countQb->resetQueryParts();
 
-        $countQb->select('count(*)')
-            ->from('('.$qb->getSQL().')', 'c');
+        // only COUNT the group by field(s) to comply with SQLs only_full_group_by mode
+        $groupBy = $qb->getQueryPart('groupBy');
+        if(!is_array($groupBy)){
+            throw new \Exception('A group by statement is required in order to get the total rows');
+        }
+        foreach ($groupBy as $field) {
+            $select = "COUNT(${field}),";
+        }
+        $countQb->add('select', substr($select,0,-1));
```

Comment:
Can you please explain to me the intent behind it? If I understand it well, it should take the last field, let's say `t.name`, wrap it in the COUNT like `COUNT(t.name)` function and then add it to the query like `COUNT(t.name` without the closing parentheses? I don't get it.

Reply:
`// only COUNT the group by field(s) to comply with SQLs only_full_group_by mode`
I think there is a mistake in the code. It was missing a `.=` Correct:
`$select .= "COUNT(${field}),";`

Reply:
So it should only remove the `,` at the end. Is it also missing a `)`?